portal-api/api-tests/v1/users/reset-password.api-test.js

```diff
@@ -122,58 +123,47 @@ describe('password reset', () => {
     });
 
     describe('/v1/users/reset-password/:token', () => {
-      it('should return error for empty current password field', async () => {
-        const { body } = await as().post({ currentPassword: '', password: '1', passwordConfirm: '1' }).to('/v1/users/reset-password/mock123');
-        expect(body.success).toEqual(false);
-        expect(body.errors.errorList.currentPassword.text).toEqual('Empty password');
-      });
-
-      it('should return error for empty new password field', async () => {
-        const { body } = await as().post({ currentPassword: '1', password: '', passwordConfirm: '1' }).to('/v1/users/reset-password/mock123');
-        expect(body.success).toEqual(false);
-        expect(body.errors.errorList.password.text).toEqual('Empty password');
-      });
-
-      it('should return error for empty new confirm password field', async () => {
-        const { body } = await as().post({ currentPassword: '1', password: '1', passwordConfirm: '' }).to('/v1/users/reset-password/mock123');
-        expect(body.success).toEqual(false);
-        expect(body.errors.errorList.passwordConfirm.text).toEqual('Empty password');
+      describe('when the user token is not valid', () => {
+        it('should return error for invalid token', async () => {
+          const { body } = await as()
+            .post({ currentPassword: 'currentPassword', password: 'newPassword', passwordConfirm: 'newPassword' })
+            .to('/v1/users/reset-password/madeUpToken123');
+          expect(body.success).toEqual(false);
+          expect(body.errors.errorList.password.text).toEqual('Password reset link is not valid');
+        });
       });
 
-      it('should return error for new passwords do not match', async () => {
-        const { body } = await as().post({ currentPassword: '123', password: '1', passwordConfirm: '2' }).to('/v1/users/reset-password/mock123');
-        expect(body.success).toEqual(false);
-        expect(body.errors.errorList.password.text).toEqual('Password do not match');
-        expect(body.errors.errorList.passwordConfirm.text).toEqual('Password do not match');
-      });
+      describe('when the user token is valid', () => {
```

Comment:
Should this be `password` instead of user?

Reply:
But the token is for a password no? Variables and params being used below are called `passwordResetToken`? Subsequent tests are testing a password is reset if the `passwordResetToken` is valid?

Reply:
good point, updated